config/initializers/01_redis.rb

```diff
@@ -3,5 +3,4 @@
 # environment specific redis host and port (see: config/redis.yml)
 REDIS_CONFIG = Rails.application.config_for(:redis).freeze
```

Comment:
```suggestion
REDIS_CONFIG = Rails.application.config_for(:redis).to_hash.freeze
```

Reply:
NBD, just my 2 cents.

Reply:
I tried it, but the way these configuration classes work is sort of goofy IMO
```
(byebug) Rails.application.config_for(:redis).class        
ActiveSupport::OrderedOptions
(byebug) Rails.application.config_for(:redis).to_hash.class
ActiveSupport::OrderedOptions
(byebug) REDIS_CONFIG[:redis].class
Rails::Application::NonSymbolAccessDeprecatedHash
(byebug) REDIS_CONFIG[:redis].to_hash.class
Hash
```